README.md

```diff
@@ -32,46 +32,41 @@ make -j4
 And that should give you `rbkit-app/RBKitClient` executable, which you can
 go ahead and run.
 
-## Qt build issue on OSX Mavericks
```

Comment:
because i think its not more there, its solved because we can either load it directly in qt creator itself or run the cmake to build. I didn't encounter any issues by using the above commands